tokio/src/task/local.rs

```diff
@@ -936,7 +936,7 @@ cfg_unstable! {
         /// [unstable]: crate#unstable-features
         /// [`Id`]: struct@crate::runtime::Id
         pub fn id(&self) -> runtime::Id {
```

Comment:
Should this method also be stable now ? Or this will be done sometime later ?

Reply:
I haven't thought about whether this method should be stable.

Can we land this PR as is and consider stabilizing this separately?

Reply:
Follow up PR: <https://github.com/tokio-rs/tokio/pull/7776>